data/human/human missions.txt

```diff
@@ -6301,3 +6301,418 @@ mission "Blind Man from Martini"
+	on complete
+		conversation
+			`You arrive on <destination>, rent a local transport, and make your way to the location specified. There's a van parked outside. When you knock on the door, Kris bursts out almost instantly. This time, they've got on a ruffly skirt, a button-down shirt, and suspenders, and their hair is in a tight bun. "Finally!" they say. "Everything's packed already; we just need to load it in and get to the spaceport."`
+			`	It takes much less time to load everything this time than the last time you helped Kris move, and soon they hop in the driver's seat of the van, calling to you, "I'll meet you at the spaceport in an hour or two!"`
+
+mission "Take Me To The South B: Followup 2"
+	name "Moving House, Again"
+	source "Lichen"
+	destination "Thule"
+	cargo "personal belongings" 4
+	passengers 1
+	to offer
+		has "Take Me To The South B: Followup 1: done"
+	on offer
+		conversation
+			`Kris' possessions fit more easily into your hold this time - it seems like they really do have less than before. Once they're loaded in and Kris climbs aboard, hoping to head off (or at least delay) their interminable tirades on the power of the state by getting some frustrations about <origin>'s ever-damp weather instead, you ask, "So what is it about <origin> that you can't stand?"`
+			`	A fire kindles in Kris' eyes, and a righteous fury seems to possess them. "I thought when I came here that I'd be among like-minded souls. There's barely any government to speak of here! But then when the war started, they were aaaaall sooooo delighted with the so-called Free Worlds! "Free"! They're just another state!"`
+			`	Kris continues in this vein for approximately an hour, apparently without taking a single breath, but you tune them out while starting your pre-takeoff checks. You've known several anarchists by this time, and Kris remains the most vocal of them, with the least coherent ideology. Fortunately, they do at least mention your destination is <destination> during their rant.`
+				accept
+	on complete
+		payment 5000 500
+		conversation
+			`You arrive at Thule, and Kris once more eagerly unloads their belongings into a local transport. As with the previous journey, they were a strange combination of genuinely polite and helpful and deeply aggravating.`
+			`	Now, though, they stand on the tarmac and take in great lungfuls of air, proclaiming it "the air of true freedom." You're somewhat more skeptical of how the locals will treat them, being an outsider, but...well, maybe they'll actually get along.`
+			`	Kris practically skips back up the ramp to hand you a credit chip for <payment> before taking their transport and heading off to whatever is awaiting them out there on <planet>.`
+
+mission "Take Me To The South C"
+	name "Stranded In Paradise"
+	minor
+	source
+		attributes "paradise"
+	destination
+		attributes "rim" "south"
+		not government "Pirate"
+		not system "Alniyat" "Atria" "Lesath" "Han"
+		not planet "Glaze" "Zug" "Bourne"
```

Comment:
No pirates because the Burnbecks are neither pirates nor particularly independent-minded.

The first "not" line comes, again, from the Pact Recon 0 mission's source line.

The planets there are ones that the Burnbecks mention in their grumbling as being out of touch, so it would be odd if they were also from there.